test/over_react/component_declaration/transformer_integration_tests/abstract_accessor_integration_test.dart

```diff
@@ -11,14 +11,14 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-library over_react.component_declaration.transformer_integration_tests.abstract_accessor_integration_test;
-
 import 'package:over_react/over_react.dart';
 import 'package:test/test.dart';
 
+// ignore: uri_has_not_been_generated
+part 'abstract_accessor_integration_test.over_react.g.dart';
+
 main() {
-  group('transformed abstract accessor integration:', () {
+  group('(dart 2) transformed abstract accessor integration:', () {
```

Comment:
What are these `(dart 2)` prefixes for?

Reply:
The prefixes should be removed. In master right now there is a duplicity on this and other component integration tests (see: [here](https://github.com/Workiva/over_react/tree/master/test/over_react/component_declaration/transformer_integration_tests)). I removed them in this branch and moved them to where the Dart1 files lived, so the diff is actually the diff between those two files. I just missed removing the prefixes.